src/skmultiflow/trees/nodes/ada_learning_node.py

```diff
@@ -10,7 +10,6 @@
 MAJORITY_CLASS = 'mc'
 NAIVE_BAYES = 'nb'
 NAIVE_BAYES_ADAPTIVE = 'nba'
```

Comment:
Please make these constants private.

Reply:
Addressed in commit [f5838ee](https://github.com/scikit-multiflow/scikit-multiflow/pull/220/commits/f5838ee9b96b27c674a13a529e065d3b91c38aa2).